modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -142,7 +164,9 @@ export default {
       editing: false,
       editingDocId: '',
       queryExtras: {},
-      holdQueries: false
+      holdQueries: false,
+      selectedItems: this.items,
+      checked: this.items.map(item => item._id)
```

Comment:
Please add a comment `// NOTE:` here for the refactor work: "`checked` is originally set in `AposTableMixin.js`"

Reply:
Done